packages/tools/pie-icons-webc/vite.config.js

```diff
@@ -30,7 +30,6 @@ export default defineConfig({
             fileName: () => 'dist/[name].js',
         },
         rollupOptions: {
-            external: [/^lit/],
```

Comment:
This wasn't doing anything wrong, it just wasn't necessary. The icons-webc package just builds components so doesn't use lit directly. I checked the icon bundle sizes and they stayed the same so should be okay to remove. I also tested in the vanilla app by removing everything except an icon and it works just fine with no other dependencies.